public/admin/config.yml

```diff
@@ -726,3 +726,59 @@ collections:
                 name: vaccinationSignupUrl
                 required: false
                 widget: string
+  - label: Vaccination Phases & Alerts
+    name: vaccinationPhasesAndAlerts
+    label_singular: Vaccination Phase
+    description: Details on vaccination phases and alerts.
+    format: json
+    files:
+      - name: Vaccine Information-State
+        label: State Vaccine Information 
+        file: src/cms-content/vaccines/state-vaccine-phases.json
+        fields:
+          - label: Regions
+            name: regions
+            widget: list
+            fields:
+              - label: Location Name
+                name: locationName
+                widget: string
+              - label: Fips Code
+                name: fips
+                widget: string
+              - label: Hidden
+                name: hidden
+                widget: boolean
+                required: false
+              - label: Eligibility Information URL
+                name: eligibilityInfoUrl
+                widget: string
+              - label: Email alert version
+                name: emailAlertVersion
+                default: 0
+                widget: number
+              - label: Phases
```

Comment:
estimated population

Reply:
Do you mean estimated size of people that would be in a phase?  Is that easy to get?

Reply:
The designs had an estimated population size for each phase, we could have this as an optional field for states that provide this info